docs/dev/developer-guides/bridging/l1-l2.md

```diff
@@ -224,3 +224,7 @@ const executeTx = await wallet.requestExecute({
 
 await executeTx.wait();
 ```
+
+## Why does `wait()` get stuck for L1->L2 transactions?
+
+If the `wait()` function takes much longer than expected, most likely the transaction has failed.
```

Comment:
```suggestion
If the `wait()` function takes a lot longer than expected, most likely the transaction has failed.
```

Reply:
**much longer** is not quite correct, use "a lot"

Reply:
Already resolved.